package.json

```diff
@@ -10,13 +10,14 @@
   "dependencies": {
     "@asyncapi/converter": "^1.1.0",
     "@asyncapi/diff": "^0.4.0",
-    "@asyncapi/generator": "^1.9.12",
-    "@asyncapi/modelina": "^0.59.9",
+    "@asyncapi/generator": "^1.9.14",
+    "@asyncapi/modelina": "^1.0.0-next.40",
     "@asyncapi/parser": "^1.17.1",
     "@asyncapi/studio": "^0.15.4",
     "@oclif/core": "^1.18.0",
     "@oclif/errors": "^1.3.5",
     "@oclif/plugin-not-found": "^2.3.1",
+    "ajv": "^8.12.0",
```

Comment:
This is a weird one, and I would be happy if someone had a different solution.

If I dont specify this dependency I keep getting the stacktrace:

```
    TypeError [ERR_INVALID_ARG_TYPE]: The "warning" argument must be of type string or an instance of Error. Received an instance of ModuleLoadError

      at Plugin.warn (node_modules/@oclif/core/lib/config/plugin.js:256:17)
      at node_modules/@oclif/core/lib/config/plugin.js:239:30
          at async Promise.all (index 7)
      at async Plugin._manifest (node_modules/@oclif/core/lib/config/plugin.js:232:24)
      at async Plugin.load (node_modules/@oclif/core/lib/config/plugin.js:127:25)
      at async Config.load (node_modules/@oclif/core/lib/config/config.js:85:9)
      at async Function.load (node_modules/@oclif/core/lib/config/config.js:79:9)
      at async Object.run (node_modules/@oclif/test/lib/load-config.js:13:26)
      at async Object.run (node_modules/@oclif/test/lib/command.js:15:30)
      at async Object.run (node_modules/fancy-test/lib/base.js:68:25)
```

After a bit of debugging, the underlying error is `Cannot find module 'ajv/dist/compile/codegen'`. This originates from the Modelina dependency, that now uses the latest parser implementation...

My hunch is that the ajv dependency are clashing between versions, but it's just weird that studio works fine 🧐 

```
% npm list ajv
@asyncapi/cli@0.27.2 /Users/lagoni/Documents/github/cli
├─┬ @asyncapi/generator@1.9.14
│ ├─┬ @asyncapi/raml-dt-schema-parser@2.0.1
│ │ └─┬ ramldt2jsonschema@1.2.3
│ │   └─┬ json-schema-migrate@0.2.0
│ │     └── ajv@5.5.2 
│ └── ajv@6.12.6 
├─┬ @asyncapi/modelina@1.0.0-next.40
│ ├─┬ @apidevtools/swagger-parser@10.1.0
│ │ └── ajv@8.12.0 
│ └─┬ @asyncapi/parser@2.0.0-next-major.11
│   ├─┬ @stoplight/spectral-core@1.15.1
│   │ └── ajv@8.11.2 
│   ├─┬ @stoplight/spectral-functions@1.7.1
│   │ └── ajv@8.11.2 
│   ├─┬ @stoplight/spectral-rulesets@1.14.1
│   │ └── ajv@8.11.2 
│   ├── ajv@8.12.0 
│   ├─┬ ajv-formats@2.1.1
│   │ └── ajv@8.11.2 
│   └─┬ webapi-parser@0.5.0
│     └── ajv@6.5.2 
├─┬ @asyncapi/parser@1.17.1
│ └── ajv@6.12.6 
├─┬ @asyncapi/studio@0.15.4
│ └─┬ @asyncapi/parser@2.0.0-next-major.10
│   └── ajv@8.11.2 
├─┬ @semantic-release/npm@7.1.3
│ └─┬ npm@7.24.2
│   └─┬ node-gyp@7.1.2
│     └─┬ request@2.88.2
│       └─┬ har-validator@5.1.5
│         └── ajv@6.12.6
``` 

So, by specifically installing this dependency it fixes everything, and I simply dont know why...